osu.Game/Overlays/Chat/ChannelControl/ControlItemMention.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+
+namespace osu.Game.Overlays.Chat.ChannelControl
+{
+    public class ControlItemMention : CircularContainer
+    {
+        private OsuSpriteText? countText;
+
+        [Resolved]
+        private BindableInt mentions { get; set; } = null!;
+
+        [Resolved]
+        private OsuColour osuColour { get; set; } = null!;
+
+        [Resolved]
+        private OverlayColourProvider colourProvider { get; set; } = null!;
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Masking = true;
+            Size = new Vector2(20, 12);
+            Alpha = 0f;
+
+            Children = new Drawable[]
+            {
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = osuColour.YellowLight,
```

Comment:
```suggestion
                    Colour = osuColour.Orange1,
```

is what is actually spec'd on design:

![2022-03-15-223747_726x241_scrot](https://user-images.githubusercontent.com/20418176/158477172-283e4aa6-1868-4afe-a7ed-27efa5259489.png)

Check out the "basic" colour theme here: https://www.figma.com/file/VIkXMYNPMtQem2RJg9k2iQ/Asset%2FColours?node-id=1838%3A3 if you haven't seen that already.